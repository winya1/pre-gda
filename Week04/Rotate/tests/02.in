24
1 10  1 1 1 1 1 1 1 1 1 1
0 10  1 2 3 4 5 6 7 8 9 10
1 10  1 2 3 4 5 6 7 8 9 10
2 10  1 2 3 4 5 6 7 8 9 10
3 10  1 2 3 4 5 6 7 8 9 10
4 10  1 2 3 4 5 6 7 8 9 10
5 10  1 2 3 4 5 6 7 8 9 10
6 10  1 2 3 4 5 6 7 8 9 10
7 10  1 2 3 4 5 6 7 8 9 10
8 10  1 2 3 4 5 6 7 8 9 10
9 10  1 2 3 4 5 6 7 8 9 10
10 10  1 2 3 4 5 6 7 8 9 10
100 10  1 2 3 4 5 6 7 8 9 10
1000 10  1 2 3 4 5 6 7 8 9 10
10000 10  1 2 3 4 5 6 7 8 9 10
100000 10  1 2 3 4 5 6 7 8 9 10
1000000 10  1 2 3 4 5 6 7 8 9 10
10000000 10  1 2 3 4 5 6 7 8 9 10
10000001 10  1 2 3 4 5 6 7 8 9 10
10000002 10  1 2 3 4 5 6 7 8 9 10
10000003 10  1 2 3 4 5 6 7 8 9 10
10000004 10  1 2 3 4 5 6 7 8 9 10
10000005 10  1 2 3 4 5 6 7 8 9 10
100000000 9  9 8 7 6 5 4 3 2 1
