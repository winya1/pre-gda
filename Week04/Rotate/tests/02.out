Case #1: 1 1 1 1 1 1 1 1 1 1
Case #2: 1 2 3 4 5 6 7 8 9 10
Case #3: 10 1 2 3 4 5 6 7 8 9
Case #4: 9 10 1 2 3 4 5 6 7 8
Case #5: 8 9 10 1 2 3 4 5 6 7
Case #6: 7 8 9 10 1 2 3 4 5 6
Case #7: 6 7 8 9 10 1 2 3 4 5
Case #8: 5 6 7 8 9 10 1 2 3 4
Case #9: 4 5 6 7 8 9 10 1 2 3
Case #10: 3 4 5 6 7 8 9 10 1 2
Case #11: 2 3 4 5 6 7 8 9 10 1
Case #12: 1 2 3 4 5 6 7 8 9 10
Case #13: 1 2 3 4 5 6 7 8 9 10
Case #14: 1 2 3 4 5 6 7 8 9 10
Case #15: 1 2 3 4 5 6 7 8 9 10
Case #16: 1 2 3 4 5 6 7 8 9 10
Case #17: 1 2 3 4 5 6 7 8 9 10
Case #18: 1 2 3 4 5 6 7 8 9 10
Case #19: 10 1 2 3 4 5 6 7 8 9
Case #20: 9 10 1 2 3 4 5 6 7 8
Case #21: 8 9 10 1 2 3 4 5 6 7
Case #22: 7 8 9 10 1 2 3 4 5 6
Case #23: 6 7 8 9 10 1 2 3 4 5
Case #24: 1 9 8 7 6 5 4 3 2
